2021-12-16 12:13:16,875 - trainer.py[line:25] - INFO: Running with identifier 20211216_121316
2021-12-16 12:13:16,875 - trainer.py[line:27] - INFO: Saving log to log\separated_gaussian\20211216_121316\run.log
2021-12-16 12:13:16,875 - trainer.py[line:28] - INFO: Using args: [('log_name', 'separated_gaussian'), ('data_train', 'dataset/separated_gaussian'), ('data_eval', None), ('output_checkpoint_dir', 'checkpoint'), ('eval_per_epoch', 20), ('log_dir', 'log'), ('tf_dir', 'tensorboard'), ('output_dir', 'output'), ('not_ignore_warning', False), ('enable_tensorboard', True), ('pack', False)]
2021-12-16 12:13:16,879 - trainer.py[line:39] - INFO: Using tensorboard, store at: tensorboard\separated_gaussian\20211216_121316
2021-12-16 12:13:16,879 - trainer.py[line:40] - INFO: Using training data: dataset/separated_gaussian
2021-12-16 12:13:16,879 - trainer.py[line:42] - INFO: Pack this file using this command if training finished: --log_dir log --log_name separated_gaussian --identifier 20211216_121316 --output_checkpoint_dir checkpoint --data_dir dataset/separated_gaussian
2021-12-16 12:13:16,879 - train.py[line:128] - INFO: Running model resnet50
2021-12-16 15:42:10,652 - trainer.py[line:54] - ERROR: [Errno 32] Broken pipe
